README.md

```diff
@@ -1,27 +1,55 @@
-Python RQL
```

Comment:
Few notes on README:
1. I don't like dependencies described in it as we have poetry and we need to have 2 places to keep in sync :)
2. Some rendering error with ##
3. Simplest example of usage of filter class is totally needed in README

![image](https://user-images.githubusercontent.com/8081667/153907069-f5d97fd4-65a5-4f19-8710-16f848bea9e6.png)

Reply:
i suggest that documentation we go over readthedocs as is being prepared.......way better in my pov.